test-cases/tests/alerts/c12-verify-3scale-uibbt-alerts.md

```diff
@@ -0,0 +1,26 @@
+---
+environments:
+  - osd-post-upgrade
+  - osd-fresh-install
```

Comment:
@b1zzu In general I would say it should be decided on the team level (in this case monitoring team) what environments their test cases should be verified against. Given that this is something that was worked on so it is more likely that some bugs might be there (compare to other areas what was not under (so heavy) development during the sprint) I would tend to think it might make sense to run on both environments. For sure we would only run on one of the (upgrade if possible) for subsequent (every third) releases.

Reply:
@trepel yes definitely the choice is on the monitoring team in this case :), just wanted to point out that is not strictly necessary to execute the test on both or all environments. We can also, as you sad, execute it the first time on both for the subsequent times only on one env, but we don't have a process for it so maybe we should add a note to not forget it

Reply:
@b1zzu this test case only targets 2.6.0, so should be fine in this case?